Sources/PackageModel/SwiftSDKs/SwiftSDK.swift

```diff
@@ -598,6 +598,82 @@ public struct SwiftSDK: Equatable {
         return nil
     }
 
+    /// Computes the target Swift SDK for the given options.
+    public static func deriveTargetSwiftSDK(
```

Comment:
Let's avoid plain `public` to avoid making SwiftPM API too broad (and we can't use `package` yet due to unresolved CI issues)

```suggestion
    @_spi(SwiftPMInternal)
    public static func deriveTargetSwiftSDK(
```

Reply:
Make sense :+1: Updated patch